web/src/features/panels/zone/util.ts

```diff
@@ -68,32 +68,27 @@ export const getZoneDataStatus = (
   return ZoneDataStatus.NO_REAL_TIME_DATA;
 };
 
-export function getContributors(zoneId: string) {
-  return {
-    zoneContributorsIndexArray: config.zones[zoneId]?.contributors as number[],
-    contributors: config.contributors,
-  };
-}
+export const getContributors = (zoneId: string) => ({
```

Comment:
Very nitpicky but we try use regular function definitions as much as possible

Reply:
I know but arrow functions are smaller when bundled basically because of this:
with functions:
```js
function one(){return "1"};
function two(){return "2"};
function three(){return "3"};
```
with arrow functions:
```js
const one=()=>"1", two=()=>"2", three=()=>"3";
```